### ISSUE TITLE
Update arguments for REST API curl commands to include accepting application/json content-type

### ISSUE THREAD

**Issue by jwitz** [astronomer/docs#228]
@ben-astro commented on [Wed May 19 2021](https://github.com/astronomer/docs-archived/issues/363)

**Affected Document:**
https://www.astronomer.io/docs/enterprise/v0.25/customize-airflow/airflow-api
https://www.astronomer.io/docs/cloud/stable/customize-airflow/airflow_api

**Severity:**

- [X] Critical: This document has incorrect or missing information. Add the `error/critical` label to this issue.
  
**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**
Reference Issue: https://github.com/astronomer/issues/issues/2695
Basically just need to add an argument to the curl command in the 2.0 API example: 
`-H "Accept:'application/json'"`

This argument already exists in our examples for experimental REST API (1st image), but doesn't exist in Airflow 2.0 REST API examples (2nd image). 
![image](https://user-images.githubusercontent.com/68117952/118852481-ae578880-b887-11eb-854f-2fc80bdf1c28.png)
![image](https://user-images.githubusercontent.com/68117952/118852500-b4e60000-b887-11eb-8230-30c486fc6f85.png)

**Additional Notes:**
